### ISSUE TITLE
How to avoid whole app crashing because of assert https http

### ISSUE THREAD

**Issue by vikmarino** [leonbreedt/FavIcon#13]
I have a large list of urls that I need to preview their favicon.
While scrolling the list some of them are like angel.co, with no http, https. 
Some contain *www*.google.com. Some are forum.site.com

While compiling in debug, the assert in Download.swift gets hit every time some non http urls comes in.
Even if I do "if !contain http -> insert http", still having this issue.

For instance this one "http://forum.tvfreeload.com/ucp.php" causes the trouble.

Console:
```
po urls
▿ 1 element
  ▿ 0 : data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAy ... rkJggg==
    - _url : data:image/png;base64,iVBO
```

Maybe you could externalise somehow the final url that needs to be downloaded so we could use our image caching libs, for example, to get and cache the final image?

**Comment by leonbreedt** [comment]
Interesting, didn't know it was possible to define icons inline like that. Fixed in 3.0.2.

**Comment by vikmarino** [comment]
Thanks!
